sdk/eventgrid/azure-eventgrid/azure/eventgrid/_consumer.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+import json
+import six
+
+from azure.core import PipelineClient
+from msrest import Deserializer, Serializer
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any
+
+from ._models import DeserializedEvent
+
+class EventGridConsumer(object):
+    """
+    A consumer responsible for deserializing event handler messages, to allow for access to strongly typed Event objects.
+    """
+
+    def deserialize_event(self, event, **kwargs):
+        # type: (Union[str, dict, bytes]) -> models.DeserializedEvent
+        """Single event in CloudEvent/EventGridEvent format will be parsed and returned as DeserializedEvent.
+        :param event: The event to be deserialized.
+        :type event: Union[str, dict, bytes]
+        :rtype: models.DeserializedEvent
+
+        :raise: :class:`ValueError`, when events are not of CloudEvent or EventGridEvent format.
+        """
+        try:
+            if isinstance(event, six.binary_type):
+                dict_event = json.loads(event.decode('ascii'))
+                deserialized_event = DeserializedEvent(dict_event)
+            elif isinstance(event, str):
+                dict_event = json.loads(event)
+                deserialized_event = DeserializedEvent(dict_event)
+            elif isinstance(event, dict):
+                deserialized_event = DeserializedEvent(event)
+        except ValueError as e:
+            print('Error: cannot deserialize event. Event does not have a valid format. Event must be a string, dict, or bytes following the CloudEvent/EventGridEvent schema.')
```

Comment:
Don't print, use logger instead

Reply:
Added `logging.basicConfig(level=logging.DEBUG)` at the top and changed `print(...)` to `_LOGGER.error(...)`